### DISCUSSION TITLE
Help needed with Blink1 Listener

### DISCUSSION THREAD

**Discussion by honey-pablo** [josephdadams/TallyArbiter#769] [Q&A]
Hi,  I'm having issues getting the blink1 python to work,  having followed the instructions here:

https://github.com/josephdadams/TallyArbiter/tree/master/listener_clients/blink1-listener

- Can anyone confirm that this code works with the latest version of TA.

I have a number of listeners and sources working with TA so I'd like to think i know how it works.

When I run the code the blink lights up white ever few seconds, which i guess reflects it's polling but not the successful double green flash expected.

The error message reports the server as being 3.1.1 which I assume means it can communicate with it (running on the same machine under docker).

Not sure what to try next! 

```
admin@blink:~ $ sudo python3 blink1-listener.py --host 127.0.0.1 --port 4455
Tally Arbiter Blink1 Listener Running. Press CTRL-C to exit.
Attempting to connect to Tally Arbiter server: 8cc254317933.:4455 (UUID 444d4667, server version 3.1.1)
Unable to connect to Tally Arbiter server: 127.0.0.1 4455
Tally Arbiter Blink1 Listener Running. Press CTRL-C to exit.
Attempting to connect to Tally Arbiter server: 8cc254317933.:4455 (UUID 444d4667, server version 3.1.1)
Unable to connect to Tally Arbiter server: 127.0.0.1 4455
^CExiting Tally Arbiter Blink1 Listener.
admin@blink:~ $
```

**Comment by cgillmor** [comment]
I'm running into the same issue when trying to run the listener script with a raspberry pi zero w. I was able to successfully run the script and connect to a server with a mac though. Not sure what the difference between the two would be. Have you had any luck solving the problem?

**Reply by honey-pablo** [reply]
No luck i'm afraid.   I also just tried it on the Mac following your comment,  strangely the previous listener script (for TA <3) does double green flash to show connection but doesn't appear as a listener.  The correct latest script for TA > 3 doesn't do anything when i run it...

`└─$ sudo python3 blink1-listener.py --host 172.17.17.21 --port 4455 --device-id camera1 --debug
Password:
Namespace(host='172.17.17.21', port='4455', device_id='camera1', debug=True, disable_reassign=False, disable_flash=False, disable_status_blink=False, skip_blink1=False)`
